src/components/Popups/ReplyField/PopupList.tsx

```diff
@@ -28,12 +32,81 @@ const options: Partial<Options> = {
     placement: 'bottom-start',
 };
 
-const PopupList = ({ reference, ...rest }: Props): JSX.Element => (
-    <PopupBase className="ba-PopupList" options={options} reference={reference} {...rest}>
-        <div className="ba-PopupList-prompt ba-PopupList-item">
-            <FormattedMessage {...messages.popupListPrompt} />
+const PopupList = ({ children, onSelect, reference, selector, ...rest }: Props): JSX.Element => {
+    const [activeItemIndex, setActiveItemIndex] = React.useState<number>(-1);
+
+    const handleKeyDown = (event: KeyboardEvent): void => {
+        const childrenCount = React.Children.count(children);
+
+        const stopDefaultEvent = (e: KeyboardEvent): void => {
```

Comment:
This function has access to `event` as a closure, so doesn't need `e` as an argument. However, it probably shouldn't be redeclared every render.